metal/resource_metal_project_api_key.go

```diff
@@ -0,0 +1,105 @@
+package metal
+
+import (
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/packethost/packngo"
+)
+
+func metalProjectAPIKey() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"project_id": {
+			Type:     schema.TypeString,
+			ForceNew: true,
+			Required: true,
+		},
+		"read_only": {
+			Type:     schema.TypeBool,
+			Default:  true,
+			ForceNew: true,
+			Optional: true,
+		},
+		"description": {
+			Type:     schema.TypeString,
+			Required: true,
+		},
+	}
+}
+
+func resourceMetalProjectAPIKey() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceMetalProjectAPIKeyCreate,
+		Read:   resourceMetalProjectAPIKeyRead,
+		Update: resourceMetalProjectAPIKeyUpdate,
+		Delete: resourceMetalProjectAPIKeyDelete,
+		Importer: &schema.ResourceImporter{
```

Comment:
`terraform import` only allows to use one parameter, usually it's resource UUID. In case of resources like this, which need 2 params - project ID and key ID, you can't really do import out-of the box like you did.

It can be solved by parsing both IDs from a single string "<project_id>:<key_id>" (https://www.terraform.io/docs/extend/resources/import.html#importer-state-function), and implementing custom Resource Importer, but I'm not sure if we need import of this resource right here right now. Maybe it's better if you just remove the importer (for the project_api_key).